Add and fix billing disclaimer (#682)
neo4j/docs-aura: modules/ROOT/pages/logging/log-forwarding.adoc

[NOTE]
====

> Aura Database and Analytics services are business-critical for our users.

Reviewer: ```suggestion
Aura Database and Analytics services are business-critical for users.
```

> We have requests to introduce more capabilities enabling access to logs and metrics to derive actionable insights using your choice of a monitoring platform.

Reviewer: ```suggestion
There are requests to introduce more capabilities enabling access to logs and metrics to derive actionable insights using your choice of monitoring platform.
```

> We have a strong roadmap of observability sharing features including security logs, query logs and other capabilities.

Reviewer: ```suggestion
Aura has a strong roadmap of observability sharing features including security logs, query logs, and other capabilities.
```

> Many of these logs can be of significant size, hence *we will introduce in the future a new consumption-based billing model including cloud egress costs*.

Reviewer: ```suggestion
Many of these logs can be of significant size, hence *a new consumption-based billing model including cloud egress costs* will be introduced in the future.
```

> We believe security is of paramount importance, hence we have decided to make security logs available for you initially at no extra charge.

Reviewer: ```suggestion
Security is of paramount importance, and therefore the availability of security logs is not a paid-for feature.
```

> We believe security is of paramount importance, hence we have decided to make security logs available for you initially at no extra charge.

Author: ```suggestion
Security is of paramount importance, and therefore the security logs is initially not a paid-for feature.
```
Suggesting to say that the feature is free, not its availability.

Reviewer: How about: 	Security is of paramount importance, and therefore the security logs are initially available for free.
